sdk/storage/azure-storage-blob/azure/storage/blob/_shared/base_client.py

```diff
@@ -69,7 +74,8 @@ def __init__(
         self,
         parsed_url,  # type: Any
         service,  # type: str
-        credential=None,  # type: Optional[Any]
+        credential= None,  # type: Optional[Union[str, Dict[str, str],
```

Comment:
Linter hasn't complained, but at the same time I don't know if this type hint is being properly parsed as a typehint (namely, line 78). Any ideas on how to check this?

Reply:
Hmmm yeah, don't love that multi-line type hint. There are two things that can check it. Intelisense in VS code and then you can generate a `.whl` and upload it to API View and see if it got it. You can just create a new review with just one iteration and search for it then delete the review when you're done.

That being said, I don't love the way it looks :). The line is too long for lint?

Reply:
Will try to make it happy with a pylint disable 😄